src/components/experiment-tracking/runs-list/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useQuery } from '../../../utils';
+
+/**
+ * Main runslist container.
+ */
+export const RunsList = () => {
+  const query = useQuery();
+
+  const run = query.get('run');
+  const compare = query.get('compare');
+  const compareList = typeof compare === 'string' ? compare.split(' ') : null;
```

Comment:
On going through the code again, I've actually taken this out into another custom hook (`useRunIdsFromUrl`) into utils that would provide the run / compare Ids from the URL - the hook would be much easier for us to access the URL runs across all components ( this could even be expanded into including all the possible query params to be exposed from this hook as well.) 

Thanks for the inspiration!